ingress/controllers/nginx/Dockerfile

```diff
@@ -14,10 +14,12 @@
 
 FROM gcr.io/google_containers/nginx-slim:0.8
 
-RUN apt-get update && apt-get install -y \
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y \
   diffutils \
+  ssl-cert \
   --no-install-recommends \
-  && rm -rf /var/lib/apt/lists/*
+  && rm -rf /var/lib/apt/lists/* \
+  && make-ssl-cert generate-default-snakeoil --force-overwrite
```

Comment:
does it have a validity param? i can easily imagine someone deploying this and it breaks mysteriously after a year. Extending validity, or loudly denouncing the use of the default cert SG (because then we can say you didn't listen, it was only meant for testing).